### DISCUSSION TITLE
How to delete the container image from local ( Alternative of docker rmi command)

### DISCUSSION THREAD

**Discussion by dasan24** [oras-project/oras#913] [Q&A]
Hello Team, Could you please provide me an idea on how to delete the image/(s) after the push is completed.
In one of our scenario, we are taking help of ORAS command line utility to pull the container images from registry and then push them after processing to another remote registry.
Now we want to know if there is any option to delete the local image which is pulled. Please let me know if there is any command in ORAS available as an alternative of _`docker rmi`_

**Comment by qweeah** [comment]
> we are taking help of ORAS command line utility to pull the container images from registry and then push them after processing to another remote registry.

How is the to-be-processed image stored locally? Can you show the used `push` command?

**Comment by TerryHowe** [comment]
`docker rmi` removes local images though, so wouldn't this request be more like the user ran `oras pull` or `oras copy` and wanted to delete maybe out of the oras cache. Using `oras copy` to oci layout that is.
